config/zones.json

```
@@ -650,7 +650,7 @@
       "hydro storage": 1308,
       "nuclear": 5943,
       "oil": 280,
-      "solar": 4788,
+      "solar": 5.898,
```

In JSON this means a little under 6 megawatts, not 6 gigawatts :sweat_smile: 

```suggestion
      "solar": 5898,
```

---

Oops, that's a very stupid mistake I made there! Thanks for pointing it out.